share/dkg/pedersen/dkg_test.go

```diff
@@ -310,6 +311,80 @@ func dkgGen() []*DistKeyGenerator {
 	return dkgs
 }
 
+func TestDistKeyShareReNew(t *testing.T){
+	fullExchange(t)
+	fullExchangeWithRenewal(t)
+
+	dkss := make([]*DistKeyShare, nbParticipants)
+	for i, dkg := range dkgs{
+		dks,_ := dkg.DistKeyShare()
+		dksReNew,_:= dkgsReNew[i].DistKeyShare()
+		dkss[i],_ = dks.Renew(dksReNew,dkg.suite)
+	}
+
+	shares := make([]*share.PriShare, nbParticipants)
+	for i, dks := range dkss {
+		shares[i] = dks.Share
+	}
+	secret, err := share.RecoverSecret(suite, shares, nbParticipants, nbParticipants) // SUMf_j(0),j:0->#participants-
+	assert.Nil(t, err)
+
+	commitSecret := suite.Point().Mul(secret, nil)
+	assert.Equal(t, dkss[0].Public().String(), commitSecret.String())
```

Comment:
We also want to check if the first public key generated during the first DKG stays the same ! Here if I'm not mistaken you are only checking if the newly shares, reconstruct to a secret that when commited (i.e. `s * G`) gives the same public point that is generated from a renewed share... I wanna know if it holds with th public point generated from the original shares !